bosond.cpp

```diff
@@ -99,125 +106,133 @@ int main(int argc, char** argv) {
     format.fmt.pix.height = height;
 
     // request desired FORMAT
-    if(ioctl(fd, VIDIOC_S_FMT, &format) < 0){
+    if (ioctl(fd, VIDIOC_S_FMT, &format) < 0) {
         perror("VIDIOC_S_FMT");
         exit(1);
     }
 
-    // we need to inform the device about buffers to use.
-    // and we need to allocate them.
-    // we’ll use a single buffer, and map our memory using mmap.
-    // All this information is sent using the VIDIOC_REQBUFS call and a
-    // v4l2_requestbuffers structure:
+    // Allocate 2 mmap buffers for retrieving video frames.
```

Comment:
remove 2 no need the number of buffers is dynamic

Reply:
Actually, there's would be quite a bit of extra work required to support for than 2 buffers, but I'll clarify this.